core/src/components/reorder-group/reorder-group.tsx

```diff
@@ -193,10 +196,21 @@ export class ReorderGroup {
     } else {
       reorderInactive();
```

Comment:
could we emit the event directly here?

```diff
     } else {
       reorderInactive();
+      this.ionItemReorder.emit({
+        from: fromIndex,
+        to: toIndex
+      });
```

Also, renamed `fromIndex` and `toIndex` to `from` and `to`.  In ionic v3 it was called that way :)